src/main/java/de/t0biii/music/domain/bStatsCostum.java

```diff
@@ -38,16 +38,6 @@ public String call() throws Exception {
             }
         }));
 
-        bstats.addCustomChart(new Metrics.SimplePie("options_printSongTitel", new Callable<String>() {
-            @Override
-            public String call() throws Exception {
-                if (printSongTitel.equalsIgnoreCase("true") || printSongTitel.equalsIgnoreCase("false")) {
-                    return printSongTitel;
-                }
-                return "unknow";
-            }
-        }));
-
```

Comment:
maybe renaming it with something like PrintJoinMessage

Reply:
Since this is the same as the one above let's discuss it in the other